standalone-node/installation_scripts/bootable-usb-prepare.sh

```diff
@@ -20,11 +20,13 @@ YELLOW='\033[0;33m'
 
 ISO="usb_files/emt-uos.iso"
 OS_IMG_PARTITION_SIZE="3000"
-K8S_PARTITION_SIZE="8000"
+K8S_PARTITION_SIZE="6000"
+GUEST_VM_SIZE="100%"
 OS_PART=5
 K8_PART=6
+USER_APPS_PART=7
```

Comment:
If partition indices are not dynamically determined, hard-coding 7 could cause issues if the partition table changes or there are device differences.